curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-data-structures-by-building-a-shopping-cart/63ee7c664f9b65137d925c8a.md

```diff
@@ -0,0 +1,321 @@
+---
+id: 63ee7c664f9b65137d925c8a
+title: Step 17
+challengeType: 0
+dashedName: step-17
+---
+
+# --description--
+
+Your `ShoppingCart` class needs the ability to add items. Create an empty `addItem` method, which takes two parameters: `id` and `products`. Creating a method might look like:
```

Comment:
Your `ShoppingCart` class needs the ability to add items. Create an empty `addItem` method, which takes two parameters: `id` and `products`. Creating a method might look like this: